packages/react/src/components/navigation/navigationRow/NavigationRow.tsx

```diff
@@ -20,9 +19,9 @@ export type NavigationRowProps = React.PropsWithChildren<{
 }>;
 
 export const NavigationRow = ({ variant = 'default', children }: NavigationRowProps) => {
-  const { dispatch } = useContext(NavigationContext);
-
-  useEffect(() => dispatch({ type: 'NAVIGATION_ROW', value: variant }), [dispatch, variant]);
+  // Can be deleted??
+  // const { dispatch } = useContext(NavigationContext);
```

Comment:
Now that we get the variant already inside the parent component, we should be able to remove this. Also we could probably change the reducer in the parent component (Navigation.tsx) to a more simple useState call, since we no longer have complex structure there?

Reply:
This is done now

Reply:
Should we handled situation where the variant is changed on the fly after first render?

Reply:
Ah good point. As it is now if variant is changed parent component won't notice it. So we actually need to update the parent component here. I can work on it

Reply:
Fixed